### DISCUSSION TITLE
expose active display in `niri msg outputs`

### DISCUSSION THREAD

**Discussion by KiaraGrouwstra** [YaLTeR/niri#361] [Ideas]
i would like to make a keybind for scaling displays. to this end, i would like to be able to check which display is in focus, such as to be able to use that info to pass to `niri msg output $OUTPUT scale 2`.
to facilitate this use-case, it would be nice if `niri msg outputs` exposed this info.

**Comment by YaLTeR** [comment]
Good idea, made an issue
